From 2314a4c1ba0b2279a90508ce77f2dfaa24700370 Mon Sep 17 00:00:00 2001
From: Omri Itach <omrii@marvell.com>
Date: Thu, 4 Feb 2016 11:24:01 +0200
Subject: [PATCH 0799/1240] defconfig: a3700: enabled sleep U-Boot commands

enabled misc u-boot commands, to enabled sleep command.

Change-Id: I8147ead8ae3319834ecaa0a814754e823196bb1a
Signed-off-by: Omri Itach <omrii@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/27221
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 configs/mvebu_armadalp_defconfig | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configs/mvebu_armadalp_defconfig b/configs/mvebu_armadalp_defconfig
index 2aa9410..1d1f904 100644
--- a/configs/mvebu_armadalp_defconfig
+++ b/configs/mvebu_armadalp_defconfig
@@ -5,10 +5,9 @@ CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_MVEBU_A3700_IO_ADDR_DEC=y
 +S:CONFIG_MVEBU_A3700_MISC_INIT=y
 +S:CONFIG_MVEBU_A3700_CLOCK=y
+CONFIG_MVEBU_MPP_BUS=y
 +S:CONFIG_MVEBU_UART_ARLP=y
 CONFIG_XENON_MMC=y
-CONFIG_MVEBU_MPP_BUS=y
-CONFIG_MVEBU_PINCTL=y
 +S:CONFIG_DEVEL_BOARD=y
 CONFIG_CMD_BDI=y
 CONFIG_CMD_BOOTD=y
@@ -20,6 +19,7 @@ CONFIG_CMD_ECHO=y
 CONFIG_CMD_NET=y
 CONFIG_CMD_PING=y
 CONFIG_CMD_MII=y
+CONFIG_CMD_MISC=y
 CONFIG_CMD_MVEBU_MPP=y
 CONFIG_CMD_MVEBU_FDT_CONFIG=y
 CONFIG_CMD_MVEBU_BUBT=y
@@ -37,6 +37,7 @@ CONFIG_PHY_MARVELL=y
 +S:CONFIG_I2C_MV=y
 +S:# CONFIG_I2C_MV_PAD_REG is not set
 CONFIG_MVEBU_SPI=y
+CONFIG_MVEBU_PINCTL=y
 CONFIG_USB=y
 CONFIG_USB_XHCI_HCD=y
 +S:CONFIG_MVEBU_COMPHY_SUPPORT=y
-- 
1.9.1

